tesseract_core/sdk/cli.py

```diff
@@ -296,9 +290,10 @@ def build_image(
                 keep_build_cache=keep_build_cache,
                 generate_only=generate_only,
             )
-    except docker.errors.BuildError as e:
-        raise UserError(f"Error building Tesseract: {e}") from e
-    except docker.errors.APIError as e:
+    except RuntimeError as e:
+        # If "build" in error message:
+        if "build" in str(e):
+            raise UserError(f"Error building Tesseract: {e}") from e
```

Comment:
Can we use a custom exception instead? I.e., define `DockerCLIException` or so that we raise whenever a command fails.